web_sheet_full_width/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,8 @@
+This module was written to extend the functionality of the web client
+to get full width in the form view sheet.
+
+This module works in community edition and in enterprise edition.
+
+This module feature and many other features and improvements to the web backend
+are included in `web_responsive module <https://github.com/OCA/web/tree/13.0/web_responsive>`_.
```

Comment:
Should this be updated to `15.0` even though OCA/web#2058 isn't merged yet?

```suggestion
are included in `web_responsive module <https://github.com/OCA/web/tree/15.0/web_responsive>`_.
```

Reply:
fixed, thank you.